examples/platforms/utils/mac_frame.h

```diff
@@ -334,6 +334,14 @@ uint8_t otMacFrameGenerateEnhAckProbingIe(uint8_t *aDest, const uint8_t *aIeData
  */
 void otMacFrameSetEnhAckProbingIe(otRadioFrame *aFrame, const uint8_t *aData, uint8_t aDataLen);
 
+/**
+ * Update the time ie in the frame.
+ *
```

Comment:
Better to mention that, if the frame doesn't contain time IE, nothing will be done.